### PR TITLE
Rearranging code to imporve readabliltiy 

### PR DESCRIPTION
**Pull request by wattale**

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 1]
```diff
@@ -2715,4 +2670,117 @@ private int addAuthenticator(Connection conn, int tenantId, String idpName,
...
+
+    /**
+     * Update the permission path for a given id
+     *
+     * @param connection    Database connection
+     * @param um_id         Id
+     * @param newPermission New permission path value
+     * @throws SQLException
+     */
+    private void updatePermissionPath(Connection connection, String um_id, String newPermission) throws SQLException {
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Please use proper camel case variable names.

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 2]
```diff
@@ -2715,4 +2670,117 @@ private int addAuthenticator(Connection conn, int tenantId, String idpName,
...
+     * @param permission Permission path
+     * @return Permission id
+     * @throws SQLException
+     */
+    private int getPermissionId(Connection connection, String permission) throws SQLException {
+        PreparedStatement selectQuery = null;
+        ResultSet resultSet = null;
+        int id = -1;
+        try {
+            selectQuery = connection.prepareStatement(ApplicationMgtDBQueries.LOAD_UM_PERMISSIONS_W);
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Variable name "selectQuery" doesn't mean anything for a prepared statement IMO. So, please use a meaningful variable name.

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 3]
```diff
@@ -2715,4 +2670,117 @@ private int addAuthenticator(Connection conn, int tenantId, String idpName,
...
+    }
+
+    /**
+     * Delete role permission mapping for a given permission id
+     *
+     * @param connection Database Connection
+     * @param entry_id   Permission id
+     * @throws SQLException
+     */
+    private void deleteRolePermissionMapping(Connection connection, int entry_id) throws SQLException {
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Use proper camel case variable names

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java [hunk 4]
```diff
@@ -2715,4 +2670,117 @@ private int addAuthenticator(Connection conn, int tenantId, String idpName,
...
+    }
+
+    /**
+     * Delete permission entry for a given id
+     *
+     * @param connection Database connection
+     * @param entry_id   Entry id
+     * @throws SQLException
+     */
+    private void deletePermission(Connection connection, int entry_id) throws SQLException {
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: Use proper camel case variable names.

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 1 above)
Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 2 above)
Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 3 above)
Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java (hunk 4 above)
Comment (wattale) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java]: fixed in the PR : https://github.com/wso2/carbon-identity/pull/1044

**Pull request merged**